theano/gof/cmodule.py

```diff
@@ -2290,13 +2290,17 @@ def print_command_line_error():
 
         if status:
             print('===============================')
+            tf = tempfile.NamedTemporaryFile(delete=False)
             for i, l in enumerate(src_code.split('\n')):
```

Comment:
Currently, `src_code` is split into lines and each line is written into temporary file. It is faster to remove this `for` loop and write `src_code` entirely in `tf`. Try: `tf.write(src_code)`